### PR TITLE
Allowing users to override php-fpm.conf.

### LINKED ISSUE #71: The builds on travis are flaky
while builds on jenkins are stable

### PR DESCRIPTION
**Pull request by tmatsuo**
Also stopped using keepalive for fcgi in nginx.conf for test stability on travis, added debug output for local docker tests.

Fixes #71 
Related to #70

### CODE REVIEW

**Comment by tmatsuo** [comment]
I think I found possible cause. The test is failing with the following error on the server:

```
  string(255) "2016/03/17 18:08:44 [error] 18#0: *5 upstream sent unexpected FastCGI record: 3 while reading response header from upstream, client: 172.17.42.1, server: , request: "GET /phpinfo.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "localhost:65080""
```

This might be a known issue with `keepalive` setting in nginx.conf.
https://github.com/perusio/drupal-with-nginx/issues/160
https://groups.drupal.org/node/283558#comment-1016818

I will remove the `keepalive` setting and see how it goes.

**Comment by tmatsuo** [comment]
Ok, the build started to pass. I squashed the commits.
@bshaffer can you take a look?

File: php-nginx/Dockerfile [hunk 1]
```diff
@@ -103,6 +103,7 @@ COPY php.ini $PHP56_DIR/lib/php.ini
 COPY php.ini $PHP7_DIR/lib/php.ini
 COPY php-fpm.conf $PHP56_DIR/etc/php-fpm.conf
 COPY php-fpm.conf $PHP7_DIR/etc/php-fpm.conf
+RUN touch $PHP56_DIR/etc/php-fpm-user.conf $PHP7_DIR/etc/php-fpm-user.conf
```
Comment (bshaffer) [inline php-nginx/Dockerfile]: Why do we `touch` these files after copying them?

**Comment by bshaffer** [comment]
I'd like to see a test, otherwise LGTM

File: php-nginx/Dockerfile (hunk 1 above)
Comment (cedricziel) [inline php-nginx/Dockerfile]: Those are the user confs, not the main ones from above. Touching them shall prevent warnings I guess.

Comment (bshaffer) [inline php-nginx/Dockerfile]: cool, thanks :+1:

Comment (tmatsuo) [inline php-nginx/Dockerfile]: Yes, just making sure the included files are there.

**Comment by tmatsuo** [comment]
Ok, added simple tests.

**Comment by bshaffer** [comment]
lookin' good

**Comment by tmatsuo** [comment]
Thanks @bshaffer 

We need to count master process + child processes. So if you limit the max_children = 1, there should be 2 processes (1 master and 1 child) in the docker image.

This time test should pass!

**Comment by tmatsuo** [comment]
Algirht, merging!

**Pull request merged**